asana/1.0.0/README.md

```diff
@@ -0,0 +1,74 @@
+# Asana Integration
+
+[Asana](https://asana.com) is a project management and task tracking platform. This page describes how to integrate Keptn with Asana.
+
+![tagged user](https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/asana/1.0.0/assets/tagged-user.png)
```

Comment:
The image cannot be found right now because you provide an absolute path referencing the main branch. Why not reference it the same way as in your [infracost PR](https://github.com/keptn-contrib/artifacthub/pull/147/files#diff-add4dc2d01912eddc751b25a1840d4315ee17893c0c4cb3bf764a2285ca5dea0R5)?

Reply:
This formatting seems to be fine since only absolute paths are currently supported (see [here](https://github.com/keptn-contrib/artifacthub#adding-images-to-the-readme-or-artifacthub-pkgyml-file)). I will leave a comment on the other PR so you can change it over there.